src/benchmarks/micro/corefx/System.Text.Json/Utf8JsonWriter/Perf.Strings.cs

```diff
@@ -0,0 +1,115 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers;
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+
+namespace System.Text.Json
```

Comment:
Most if not all CoreFx benchmarks are under `System.*.Tests` namespace instead of under the same namespace of the feature. I will suggest to follow that pattern too.

Reply:
Doesn't look like it. I was following precedent:
https://github.com/dotnet/performance/blob/c5e5f7e8a2eaccecb7752d1eab51c143adcd742f/src/benchmarks/micro/corefx/System.Memory/Span.cs#L12
https://github.com/dotnet/performance/blob/3941f11e000edd7a404599fc2e401129c25bc4ad/src/benchmarks/micro/corefx/System.Collections/Create/CtorDefaultSizeNonGeneric.cs#L8
https://github.com/dotnet/performance/blob/0c626c0de72e9e018bb02d0f92b97117098f1fee/src/benchmarks/micro/corefx/System/Hashing.cs#L9

But if you think we should always have `.Tests` suffix, I don't mind updating it.

Reply:
I thought we'd published guidance about this but I can't find it. I do recall Adam saying something about it.

Reply:
Yes, not every merged commit has been consistent, but moving forward we should fix this.